docs/running-nio/local-pubkeeper.md

```diff
@@ -0,0 +1,58 @@
+# Running a Local Pubkeeper Server
+
+Sometimes you don't want to have to connect to the managed Pubkeeper server in the cloud but rather run and connect to one on your local network. This allows for the design of fully offline systems that do not require an internet connection. 
+
+## Installation
+
+If your nio license supports it, you may download a Python wheel from your [binary download page](https://account.n.io/binaries/download). This wheel contains the necessary libraries to run the Pubkeeper server.
```

Comment:
I would make it clear here that this is a 'Pubkeeper server Python wheel' or 'Pubkeeper server binary.' Maybe: "you many download a Pubkeeper server Python wheel when you select the Pubkeeper server on the binary download page."